aws/aws/lib/lambdas/syncBlockProd.mjs

```diff
@@ -62,8 +62,10 @@ exports.handler = async event => {
           historicalData: filteredData.slice(0, Math.min(MAX_L2_BLOCKS, filteredData.length)),
           maxBlock,
           isLast: !dataBlockNext?.Item,
-          hashBlockLast: dataBlockLast?.Item?.blockHash ?? null,
-          hashBlockNext: dataBlockNext?.Item?.blockHash ?? null,
+          rootBlockLast: dataBlockLast?.Item?.block?.root ?? null,
```

Comment:
Why we change from `hashBlockLast` and `hashBlockNext` to `rootBlockLast` and `rootBlockNext`?
Is not enough by keeping the previous version with the block hash and adding `numberBlockLast` and `numberBlockNext`?

Reply:
Hi David. Good question. Unfortunately the information about `hash` by itself is not enough to compare two block in wallet. We store only `root` in `timber` table that allows me to associate `timber` with the `block`. Thats why I need this info from the backend.